Dockerfile

```diff
@@ -4,20 +4,20 @@ RUN apt-get update && apt-get install --no-install-recommends -y git \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-RUN pip install pipenv==2018.11.26
+RUN pip install poetry
```

Comment:
We should pin poetry, as that is recommended when working with docker files. We did this for pipenv, we should do the same for poetry.